services/web/server/src/simcore_service_webserver/statics/_events.py

```diff
@@ -118,6 +121,19 @@ async def create_and_cache_statics_json(app: web.Application) -> None:
         if (p := product.login_settings) and (v := p.get("LOGIN_2FA_REQUIRED", None)):
             data["webserverLogin"].update({"LOGIN_2FA_REQUIRED": v})
 
+        # replace vcsReleaseUrl with curated release url
+        vtag = app_settings.SIMCORE_VCS_RELEASE_TAG
+        if (
+            vtag
+            and re.match(_RE_PRODUCTION_RELEASE_VERSION, vtag)
+            and product.vendor
+            and (template_url := product.vendor.get("release_notes_url_template", None))
+            and template_url
```

Comment:
you do not need the last `and template_url` .  the previous will evalute true/false